stdlib/public/Darwin/Accelerate/vDSP_FFT.swift

```diff
@@ -330,7 +330,7 @@ struct vDSP_FFTFunctions {
 }
 
 
-@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+@available(swift, deprecated: 5.2, message: "Use the `withUnsafeMutableBufferPointer` method on the real and imaginary arrays to create `DSPSplitComplex` for a defined scope.")
```

Comment:
I don't think these deprecations should be versioned. (They definitely shouldn't be tied to the compiler/language version, because the language mode is overridable.)

We'll also need to keep original availability information, since otherwise we could break back deployment for people who decide to still call these.

I think this would be the correct incantation:

```
@available(macOS, introduced: 10.15, deprecated)
@available(iOS, introduced: 13.0, deprecated)
@available(watchOS, introduced: 6.0, deprecated)
@available(tvOS, introduced: 13.0, deprecated)
@available(*, deprecated)
extension Foo { ... }
```

We need to figure out where the deprecation message goes. I sure hope we don't have to repeat it five times...

Reply:
Hi! 

I've added the message five times, let me know what you think.

Cheers.

Reply:
Looks good! Shame about the repetitions. 🙁